core/models.py

```diff
@@ -192,6 +213,78 @@ def delete(self):
         self.is_deleted = True
         self.save()
 
+    def add_default_content(self):
+        """Populate EventPageContent with default layout"""
+        data = get_default_eventpage_data()
+
+        for i, section in enumerate(data):
+            section['position'] = i
+            section['content'] = render_to_string(section['template'])
+            del section['template']
+            self.content.create(**section)
+
+    def add_default_menu(self):
+        """Populate EventPageMenu with default links"""
+        data = get_default_menu()
+
+        for i, link in enumerate(data):
+            link['position'] = i
+            self.menu.create(**link)
+
+    def invite_organizer_to_team(self, user, is_new_user, password):
+        self.team.add(user)
+        if is_new_user:
+            errors = []
+            try:
+                user.invite_to_slack()
+            except (ConnectionError, SlackerError) as e:
+                errors.append(
+                    'Slack invite unsuccessful, reason: {}'.format(e)
+                )
+            notify_new_user(user, event=self, password=password, errors=errors)
+        else:
+            notify_existing_user(user, event=self)
+
+    def add_organizer(self, email, first_name, last_name):
+        """
+            Add organizer to the event.
+
+            TODO: we need to think if create_organizers and create_events
+            are the best place for these logic. Maybe we should move it back to
+            the models.
+        """
+        defaults = {
+            "first_name": first_name,
+            "last_name": last_name,
+            "is_staff": True,
+            "is_active": True
+        }
+        user, created = User.objects.get_or_create(
+            email=email,
+            defaults=defaults
+        )
+        password = None
+        if created:
+            password = user.generate_password()
+            user.add_to_organizers_group()
+
+        self.invite_organizer_to_team(user, created, password)
+        return user
+
+    def create_organizers(self, team, email_password):
```

Comment:
Add organizer totally belongs here as it can be reaused on an instance `create_organizers` might find a better place.

I think we could just create a file called something like initial_event_data and put the functions there. They might be called just like from here. I somehow feel that putting non-reusable methods later available on the model instance is not the best idea :)